src/main/java/org/candlepin/model/EntitlementCurator.java

```diff
@@ -562,6 +562,51 @@ private int countMatchesByFilters(AbstractHibernateObject<?> object, String obje
         return this.cpQueryFactory.buildQuery(this.currentSession(), criteria);
     }
 
+    /**
+     * Fetches a the entitlements used by consumers in the specified environment.
```

Comment:
Also, let's fix this comment to read better, and also mention that it's fetching based on content ids (in addition to environment)